checker/src/main/java/org/checkerframework/checker/nonempty/NonEmptyTransfer.java

```diff
@@ -68,6 +69,9 @@ public TransferResult<CFValue, CFStore> visitEqualTo(
             EqualToNode n, TransferInput<CFValue, CFStore> in) {
         TransferResult<CFValue, CFStore> result = super.visitEqualTo(n, in);
 
+        // Expand the input store into conditional store if it's regular store
```

Comment:
```suggestion
        // Expand the input store into a conditional store if it is a regular store.
```

Try to avoid contractions. Also end comments with a period.